package.json

```diff
@@ -29,6 +29,7 @@
         "express": "^4.17.1 ",
```

Comment:
https://github.com/navikt/rekrutteringsbistand-stilling/network/alerts
Upgrade serialize-javascript to version 2.1.1 
Upgrade lodash.template to version 4.5.0 or later.